Tasks/VsTestV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -22,6 +22,8 @@
   "loc.input.help.tcmTestRun": "Test run based selection is used when triggering automated test runs from the test hub. This option cannot be used for running tests in the CI/CD pipeline.",
   "loc.input.label.searchFolder": "Search folder",
   "loc.input.help.searchFolder": "Folder to search for the test assemblies.",
+  "loc.input.label.resultsDirectory": "Test results directory",
+  "loc.input.help.resultsDirectory": "Folder to store test results. The vstest task cleans up results directory before test run. $(Agent.TempDirectory) is cleaned after each pipeline run. In case the default value is overriden, the directory will not be automatically cleaned. When results directory is provided in both task and runsettings, the runsettings will be given priority.",
```

Comment:
Folder to store test results. When this input is not specified, results are stored in $(Agent.TempDirectory) by default, which is cleaned at the end of a pipeline run. 
When the default value is overriden, the specified folder will be cleaned by the task before tests are run.